applications/FluidDynamicsApplication/custom_strategies/strategies/residualbased_predictorcorrector_velocity_bossak_scheme_turbulent.h

```diff
@@ -813,81 +783,67 @@ namespace Kratos {
          */
         void PeriodicConditionProjectionCorrection(ModelPart& rModelPart)
```

Comment:
This appears the same as for the gear scheme (which I commented heavily). It should be refactored to eliminate the duplicate code.

Reply:
I duplicated the new code here too (:-1:). The refactoring needed to merge both classes would be significant and I'd prefer not to do it in this PR. Anyways, the plan is to clean up the fluid schemes "soon" to provide better support for the BDF one and bring them to the level of the newer ones on the Structural application and the core.

Reply:
ok